src/renderer/system/utils.js

```diff
@@ -44,10 +44,25 @@ export function readClipboard() {
   return clipboard.readText();
 }
 
+export function isUrl(url) {
+  var pattern = new RegExp(
+    '^(https?:\\/\\/)?' +
+      '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|' +
+      '((\\d{1,3}\\.){3}\\d{1,3}))' +
+      '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' +
+      '(\\?[;&a-z\\d%_.~+=-]*)?' +
+      '(\\#[-a-z\\d_]*)?$',
+    'i'
+  );
+
+  return !!pattern.test(url);
+}
+
 export function openUrl(url) {
-  if (!/^http(s)?:\/\//i.test(url)) {
+  if (!isUrl(url)) {
```

Comment:
I would suggest reverting this functionality, as it's checking for a protocol. Might be best to use `^\w+:\/\/` or something else, and if it's false, assume `https`. The protocol is required for opening URLs, and we should default to https for security.

Reply:
Yes, you're right. I'll revert it.

Reply:
Fixed.